pkg/workloads/console/controller.go

```diff
@@ -213,6 +218,32 @@ func (r *reconciler) createOrUpdate(expected reconcile.ObjWithMeta, kind string,
 	return nil
 }
 
+// Ensure the console timeout is between [1, template.MaxTimeoutSeconds]
+func (r *reconciler) clampTimeout(template *workloadsv1alpha1.ConsoleTemplate) error {
+	var timeout int
+	switch {
+	case r.console.Spec.TimeoutSeconds < 1:
+		timeout = template.Spec.DefaultTimeoutSeconds
+	case r.console.Spec.TimeoutSeconds > template.Spec.MaxTimeoutSeconds:
+		timeout = template.Spec.MaxTimeoutSeconds
```

Comment:
When decreasing the specified timeout to the max timeout, it would be good to emit a warning event, so that we can make the user aware. Otherwise we're just silently mutating what was submitted.

But let's tackle this when we come to sort out events!